### DISCUSSION TITLE
Could obs be used for local streaming?

### DISCUSSION THREAD

**Discussion by qleguennec** [obsproject/obs-studio#6822] [Development Questions] [answered]
Hello,

We would like to use obs to locally stream a videogame running in the client's computer into our website. Everything should be local, with the web page listening to obs in localhost.

Would obs the right tool for that task?

**Comment by Fenrirthviti** [comment, marked as answer]
If you host a local RTMP server, sure. Discussions aren't really the right place to ask for support, however. I'd recommend joining the discord and asking in #tech-talk or making a post on our forums for further assistance.
